### PR TITLE
Remove Android/iOS comments

### PR DESCRIPTION
**Pull request by xsacha**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -65,12 +64,10 @@ matrix:
       env: PROJECT_DIR=examples/CURL TOOLCHAIN=clang-libstdcxx
     - os: osx
       env: PROJECT_DIR=examples/CURL TOOLCHAIN=osx-10-9
```
Comment (ruslo) [inline .travis.yml]: Travis images for OSX changed recently, defaults now osx-10-11 and ios-9-3. I've pushed update of configuration, please rebase this one too.

**Comment by xsacha** [comment]
Failure: https://travis-ci.org/ingenue/hunter/jobs/169389003
Has the old error about HAVE_POLL_FINE_EXITCODE not being set. Code not updated?

**Comment by ruslo** [comment]
> Has the old error about HAVE_POLL_FINE_EXITCODE not being set. Code not updated?

Yes, rebase please.

**Comment by xsacha** [comment]
CURL fails on Mac cross-compiling Android because OpenSSL fails.
CURL fails on Mac cross-compiling iOS because it should be POSIX instead of GLIBC strerror (pull req?)

**Comment by ruslo** [comment]
> CURL fails on Mac cross-compiling iOS because it should be POSIX instead of GLIBC strerror (pull req?)

What? :)

Anyway ingenue/hunter pull requests is only for .travis.yml/appveyor.yml configurations so if there are some fixes need to be introduced it should be done in other place.

**Comment by xsacha** [comment]
I don't have a Mac so I'd like to test it here first before pullreq to hunter.

**Comment by xsacha** [comment]
Updated.
Didn't bother fixing the Mac case sorry :(

**Comment by ruslo** [comment]
> Didn't bother fixing the Mac case sorry :(

No problems, it's okay to fix only what you can test. You don't have to fix everything, it's not practical. If Travis/AppVeyor tests work then it means old code is not broken, it's enough.

> Updated

Thanks.

**Comment by ruslo** [comment]
Uploading:
- https://ci.appveyor.com/project/ingenue/hunter/build/1.0.1120
- https://travis-ci.org/ingenue/hunter/builds/170006512

**Pull request merged**